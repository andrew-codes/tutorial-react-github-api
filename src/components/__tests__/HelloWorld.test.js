import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import HelloWorld from './../HelloWorld';

test('it renders a Hello World welcome message', () => {
    const wrapper = shallow(
        <HelloWorld/>
    );
    const snapshot = toJson(wrapper);
    expect(snapshot).toMatchSnapshot();
});
