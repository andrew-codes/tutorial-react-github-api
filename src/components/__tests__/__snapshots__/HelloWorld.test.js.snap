// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders a Hello World welcome message 1`] = `
<h1>
  Hello World
</h1>
`;
